Log file can not be found for multiproject Gradle projects

### Versions used 

Akka version: 
akka-grpc-gradle-plugin:0.7.3

### Expected Behavior

When building a Gradle project with multiple subprojects (e.g., a root gradle build and at least one subproject build), building the project from the root project should work.

Specifically: `./gradlew build` from the root project's directory should complete successfully.

The same goes for if the project is built not from the root project, but from the specific subproject.

### Actual Behavior

If the build is triggered from the **_subproject_**, things work as expected for that subproject.

If the build is triggered from the **_root_**, the Gradle plugin complains that a log does not exist (see below for relevant logs).

This regression is due to a change made in PR #674, which addresses issue #669.

### Relevant logs

```
> Task :subProject:generateProto FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':subProject:generateProto'.
> protoc: stdout: . stderr: /Users/eorendain/Documents/myProject/subProject/src/main/protobuf: warning: directory does not exist.
  /Users/eorendain/Documents/myProject/subProject/app/protobuf: warning: directory does not exist.
  /Users/eorendain/Documents/myProject/subProject/app/proto: warning: directory does not exist.
  bytestream.proto: warning: Import google/protobuf/wrappers.proto but not used.
  Exception in thread "main" java.io.FileNotFoundException: build/akka-grpc-gradle-plugin.log (No such file or directory)
        at java.base/java.io.FileOutputStream.open0(Native Method)
        at java.base/java.io.FileOutputStream.open(FileOutputStream.java:298)
        at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:237)
        at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:187)
        at java.base/java.io.PrintWriter.<init>(PrintWriter.java:208)
        at java.base/java.io.PrintWriter.<init>(PrintWriter.java:248)
        at akka.grpc.gen.FileLogger.<init>(Logging.scala:35)
        at akka.grpc.gen.Main$.$anonfun$logger$1(Main.scala:48)
        at scala.Option.map(Option.scala:230)
        at akka.grpc.gen.Main$.delayedEndpoint$akka$grpc$gen$Main$1(Main.scala:48)
        at akka.grpc.gen.Main$delayedInit$body.apply(Main.scala:14)
        at scala.Function0.apply$mcV$sp(Function0.scala:39)
        at scala.Function0.apply$mcV$sp$(Function0.scala:39)
        at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)
        at scala.App.$anonfun$main$1$adapted(App.scala:80)
        at scala.collection.immutable.List.foreach(List.scala:392)
        at scala.App.main(App.scala:80)
        at scala.App.main$(App.scala:78)
        at akka.grpc.gen.Main$.main(Main.scala:14)
        at akka.grpc.gen.Main.main(Main.scala)
  --akkaGrpc_out: protoc-gen-akkaGrpc: Plugin failed with status code 1.
```

### Reproducible Test Case

Apologies, I don't have much time to whip up a test case, but one would be simple to create.  Simply create a multi-project Gradle build wherein a subproject uses this Akka gRPC Gradle plugin, and run the commands as outlined above.


### Fix

I'll submit a PR with a fix in a few moments for testing/feedback.

--- a/codegen/src/main/scala/akka/grpc/gen/Main.scala
+++ b/codegen/src/main/scala/akka/grpc/gen/Main.scala
@@ -5,6 +5,7 @@
 package akka.grpc.gen
 
 import java.io.ByteArrayOutputStream
+import java.net.URLDecoder
 
 import com.google.protobuf.compiler.PluginProtos.CodeGeneratorRequest
 import akka.grpc.gen.javadsl.{ JavaClientCodeGenerator, JavaInterfaceCodeGenerator, JavaServerCodeGenerator }
@@ -45,7 +46,8 @@ object Main extends App {
   private val extraGenerators: List[String] =
     parameters.getOrElse("extra_generators", "").split(";").toList.filter(_ != "")
 
-  private val logger = parameters.get("logfile").map(new FileLogger(_)).getOrElse(SilencedLogger)
+  private val logger: Logger =
+    parameters.get("logfile_enc").map(URLDecoder.decode(_, "utf-8")).map(new FileLogger(_)).getOrElse(SilencedLogger)
 
   val out = {
     val codeGenerators =

--- a/codegen/src/main/scala/akka/grpc/gen/Main.scala
+++ b/codegen/src/main/scala/akka/grpc/gen/Main.scala
@@ -46,8 +46,14 @@ object Main extends App {
   private val extraGenerators: List[String] =
     parameters.getOrElse("extra_generators", "").split(";").toList.filter(_ != "")
 
+  // Prefer logfile_enc with fallback to logfile
   private val logger: Logger =
-    parameters.get("logfile_enc").map(URLDecoder.decode(_, "utf-8")).map(new FileLogger(_)).getOrElse(SilencedLogger)
+    parameters
+      .get("logfile_enc")
+      .map(URLDecoder.decode(_, "utf-8"))
+      .orElse(parameters.get("logfile"))
+      .map(new FileLogger(_))
+      .getOrElse(SilencedLogger)
 
   val out = {
     val codeGenerators =

--- a/gradle-plugin/src/main/groovy/akka/grpc/gradle/AkkaGrpcPlugin.groovy
+++ b/gradle-plugin/src/main/groovy/akka/grpc/gradle/AkkaGrpcPlugin.groovy
@@ -11,6 +11,7 @@ import org.gradle.internal.component.local.model.DefaultProjectComponentSelector
 import org.gradle.util.GradleVersion
 import org.gradle.util.VersionNumber
 
+import java.net.URLEncoder
 import java.nio.file.Files
 import java.nio.file.Path
 
@@ -125,7 +126,7 @@ class AkkaGrpcPlugin implements Plugin<Project> {
                             option "server_power_apis=${akkaGrpcExt.serverPowerApis}"
                             option "use_play_actions=${akkaGrpcExt.usePlayActions}"
                             option "extra_generators=${akkaGrpcExt.extraGenerators.join(';')}"
-                            option "logfile=${project.projectDir.toPath().relativize(logFile).toString()}"
+                            option "logfile_enc=${URLEncoder.encode(logFile.toString(), "utf-8")}"
                             if (akkaGrpcExt.includeStdTypes) {
                                 option "include_std_types=true"
                             }

